examples/fodo_userdef/README.rst

```diff
@@ -0,0 +1,85 @@
+.. _examples-fodo-userdef:
+
+User-Defined Linear Element
+===========================
+
+This implements the same FODO cell as the :ref:`stable FODO cell example <examples-fodo>`.
+However, in the example here we define *additional user-defined, custom linear elements* by providing a custom matrix.
```

Comment:
TODO: In the analysis of the FODO script, we should compare also values of the reference particle, in particular `s` and energy.